projects/media-viewer/src/lib/annotations/annotation.component.html

```diff
@@ -1,18 +1,18 @@
-<div class="annotation-wrapper">
+<div class="annotation-wrapper" tabindex="1" (focusout)="toggleSelection(false)">
```

Comment:
focusout only works in input/textarea fields - if you select the title of the comment then try to click somewhere else it probably won't unselect